Update glyphs

rettinghaus opened the pull request:
This PR updates some names in the `supported.xsl`. 
As far as I can see nobody needs reversed or turned clefs right now, so I removed them.

lpugin commented:
Any particular issue with reversed clefs? It would be better to leave some in case somebody uses them.

rettinghaus commented:
No issue, I'm just pretty sure I switched them on when I added the glyphs in Leipzig, although they weren't needed anywhere. I leave the extracted glyphs in for now.

lpugin commented:
I guess we also need not to remove them from `supported.xsl` otherwise they will not be usable any more.
